WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java

```diff
@@ -1447,39 +1631,41 @@ private void onCommentLiked(OnCommentChanged event) {
     @Subscribe(threadMode = ThreadMode.MAIN)
     @OptIn(markerClass = DelicateCoroutinesApi.class)
     public void onCommentChanged(OnCommentChanged event) {
-        setProgressVisible(false);
-        // requesting local comment cache refresh
-        BuildersKt.launch(GlobalScope.INSTANCE,
-                Dispatchers.getMain(),
-                CoroutineStart.DEFAULT,
-                (coroutineScope, continuation) -> mLocalCommentCacheUpdateHandler.requestCommentsUpdate(continuation)
-
-        );
-        // Moderating comment
-        if (event.causeOfChange == CommentAction.PUSH_COMMENT) {
-            onCommentModerated(event);
-            mPreviousStatus = null;
-            return;
-        }
+        if (mBinding != null && mReplyBinding != null && mActionBinding != null && mSite != null && mComment != null) {
+            setProgressVisible(mBinding, false);
+            // requesting local comment cache refresh
+            BuildersKt.launch(GlobalScope.INSTANCE,
+                    Dispatchers.getMain(),
+                    CoroutineStart.DEFAULT,
+                    (coroutineScope, continuation) -> mLocalCommentCacheUpdateHandler.requestCommentsUpdate(
+                            continuation
+                    )
+            );
```

Comment:
Although this might be an edge case, I still think this is a good catch and we better be safe than sorry, thanks for this comment @irfano , done here: bc8ca00904fdad5dba82c2f74a5d714b95554bdf